### ISSUE TITLE
Update `@yamada-ui/carousel` accessibility

### ISSUE THREAD

**Issue by setodeve** [yamada-ui/yamada-ui#1728]
### Description
This issue aims to address accessibility deficiencies in our `carousel component` as per W3C guidelines. 
The component currently does not meet the accessibility standards outlined in the W3C documentation. 

The goal is to ensure the component is accessible to all users, including those using assistive technologies.
Ensure that all unchecked items in this checklist should be addressed.

#### General Requirements
- **Visible Labels:**
  - [ ] Carousel container has an accessible label using `aria-labelledby` or `aria-label`.
  - [x] Label does not include the word "carousel".

- **Roles and Descriptions:**
  - [ ] Carousel container has `role="region"` or `role="group"`.
  - [ ] `aria-roledescription="carousel"` is set on the carousel container.
  - [ ] Each slide container has `role="group"` with `aria-roledescription="slide"`.

#### Rotation Control
- **Rotation Control Button:**
  - [x] Exists as a native button element or implements the button pattern.
  - [ ] Accessible label provided by `aria-label` or inner text.
  - [ ] Label changes based on action (e.g., "Stop slide rotation" or "Start slide rotation").
  - [x] Button does not use `aria-pressed`.

#### Slide Navigation
- **Next and Previous Slide Controls:**
  - [x] Exist as native button elements or implement the button pattern.
  - [x] Styled as arrows.
  - [ ] Do not move focus when activated.

- **Slide Picker Controls (if present):**
  - [x] Group of elements, often styled as small dots or tabs.
  - [x] Each control is a native button element or implements the button pattern.
  - [ ] Accessible name indicates the slide it will display.
  - [ ] Current slide control has `aria-disabled="true"`.

#### Keyboard Interaction
- **Tab and Shift + Tab Navigation:**
  - [x] Moves focus through carousel interactive elements.

- **Automatic Slide Rotation:**
  - [ ] Stops when any carousel element receives keyboard focus.
  - [ ] Does not resume unless the user activates the rotation control.
  - [x] Stops when the mouse hovers over the carousel.

#### WAI-ARIA Implementation
- **Basic Carousel Elements:**
  - [ ] Carousel container includes all components, with `role="region"` or `role="group"`.
  - [ ] Rotation, next slide, and previous slide controls are native button elements or implement the button pattern.
  - [ ] Each slide container has `role="group"` with `aria-roledescription="slide"`.
  - [ ] Each slide has an accessible name using `aria-labelledby` or `aria-label`.
  - [ ] Wrapper element for slides has `aria-atomic="false"` and `aria-live="off"` if rotating, `aria-live="polite"` if not.

- **Tabbed Carousel Elements:**
  - [ ] Slide containers have `role="tabpanel"` instead of `role="group"`.
  - [ ] Slide picker controls follow the Tabs Pattern.
  - [ ] Each control is a tab element, displaying the associated slide when activated.
  - [ ] Accessible name of each tab indicates the slide it will display.
  - [ ] Controls are grouped in a `tablist` element with an accessible name.

- **Grouped Carousel Elements:**
  - [ ] Slide picker controls are in an element with `role="group"`.
  - [ ] Group containing picker controls has an accessible label.
  - [ ] Each picker control is a native button element or implements the button pattern.
  - [ ] Accessible name of each picker button matches the slide it displays.
  - [ ] Current slide picker button has `aria-disabled="true"`.

### Problem Statement/Justification

none

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

### References

https://www.w3.org/WAI/ARIA/apg/patterns/carousel/

**Comment by koralle** [comment]
I try it.

**Comment by uraaaa24** [comment]
> Rotation Control Button:
 Exists as a native button element or implements the button pattern.
 Accessible label provided by aria-label or inner text.
 Label changes based on action (e.g., "Stop slide rotation" or "Start slide rotation").
 Button does not use aria-pressed.

In Yamada UI, the Rotation Control Button does not exist, so it is not included.

**Comment by uraaaa24** [comment]
**Next and Previous Slide Controls:**

> Exist as native button elements or implement the button pattern.

No problem because it uses Yamada UI's Button component internally.
Button component is compliant with native button elements.

> Styled as arrows.

<img width="72" alt="image" src="https://github.com/user-attachments/assets/58c61d1f-d935-4c3b-894b-60ba9ef116f9">
<img width="72" alt="image" src="https://github.com/user-attachments/assets/0f1557a0-b7bb-4b7c-8e0f-582f0efde0a7">

No problem because of the use of arrow icons.

> Do not move focus when activated.
 
https://github.com/user-attachments/assets/06c899e4-ee20-405f-9682-cd15d22e760f

No problem, because after pressing the prev or next button, I can press the tab and then press the button as it is.

**Comment by hirotomoyamada** [comment]
@uraaaa24 

It is possible to control the rotation of the slide.Currently, the rotation stops when the pointer is moved over the slide.

I also feel that it was possible for the developer to place arbitrary buttons to stop the slide.
